adapters/rubicon.go

```diff
@@ -216,13 +216,17 @@ func (a *RubiconAdapter) callOne(ctx context.Context, req *pbs.PBSRequest, reqJS
 func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
 	requests := make([]bytes.Buffer, len(bidder.AdUnits))
 	for i, unit := range bidder.AdUnits {
-		rubiReq := makeOpenRTBGeneric(req, bidder, a.FamilyName())
+		rubiReq, err := makeOpenRTBGeneric(req, bidder, a.FamilyName(), []pbs.MediaType{pbs.MEDIA_TYPE_BANNER}, true)
+		if err != nil {
+			continue
+		}
+		// TODO: Implement special video handling for Rubicon
```

Comment:
TODO messages don't have a ton of value on their own... I think we should either:

1. Do it in this PR
2. Log a Github issue and leave it in the comment, like `TODO(#issueNum): ...`
3. Delete the TODO

Not sure which fits best here.

Reply:
Frankly, I don't even remember what is it about. I will remove it.